naples/routes/subscription.py

```diff
@@ -147,18 +151,19 @@ async def webhook_received(
 
     event_type = event["type"]
 
-    subscription = event_data["object"]
-
-    db_subscription: m.Subscription | None = get_user_subscription(subscription["customer"], db)
-
-    if not db_subscription:
-        log(log.ERROR, "User subscription not found")
-        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="User subscription not found")
-
     if event_type == "checkout.session.completed":
         session = event_data["object"]
```

Comment:
why do we use plain python objects? al plain python objects should be wrapped in pydantic schemas!